src/Runner.js

```diff
@@ -184,7 +184,9 @@ function run(transformFile, paths, options) {
             contents += d.toString();
           })
           .on('end', () => {
-            temp.open('jscodeshift', (err, info) => {
+            const re = /(?:\.([^.]+))?$/;
+            const ext = re.exec(transformFile)[1];
```

Comment:
As far as I can tell, you should be able to use [`path.extname`](https://nodejs.org/api/path.html#path_path_extname_path) here. Then you don't have to deal with regular expressions.